k2.ipynb

```diff
@@ -0,0 +1,858 @@
+{
```

Comment:
do we need to include this file? seems a lot of logs in the file? I noticed that you have add the corresponding link of `Google Colab` in `README.md`

Reply:
I've deleted it and changed the link in README.md